nav2_amcl/src/amcl_node.cpp

```diff
@@ -1334,11 +1334,6 @@ AmclNode::initTransforms()
 
   // Initialize transform listener and broadcaster
   tf_buffer_ = std::make_shared<tf2_ros::Buffer>(get_clock());
-  auto timer_interface = std::make_shared<tf2_ros::CreateTimerROS>(
```

Comment:
Question: is removing this really necessary? I don't think this as anything to do with the thread spinning/new node. This is used in  `waitForTransform` which is used under the hood in many TF calls. I think this should be reverted unless you know something I'm missing.